v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -292,4 +292,23 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   Options.RpcPriority getSpannerPriority();
 
   void setSpannerPriority(Options.RpcPriority value);
+
+  @TemplateParameter.Text(
+      order = 24,
+      optional = true,
+      description =
+          "Hint for number of uniformization stages. Currently Applicable only for jdc based sources like MySql or PG. Leave 0 or default to disable uniformization. Set to -1 for a log(numPartition) number of stages.",
```

Comment:
nit: PG -> PostgreSQL

Reply:
Done. Thanks.
We are adding options to let user configure the uniformization stages. They are set to 0 by default to avoid the overhead of counting in most of the typical use cases where primary key space is fairly uniform.